deps/v8/gypfiles/v8.gyp

```diff
@@ -130,6 +130,7 @@
               'v8_enable_verify_predictable=<(v8_enable_verify_predictable)',
               'v8_target_cpu=<(v8_target_arch)',
               'v8_use_snapshot=<(v8_use_snapshot)',
+              'v8_use_siphash=<(v8_use_siphash)',
```

Comment:
Ohh. We've been missing this with some of the other V8 new features...
/me writes himself a TODO note